### ISSUE TITLE
`rollup-plugin-babel` doesn't understand babel sourcemaps

### ISSUE THREAD

**Issue by cprecioso** [rollup/rollup-plugin-babel#49]
[Example reproduction](https://gist.github.com/cprecioso/3e21060f2de92cbc51f5bfb70df7212b)

Basically what it says in the title. The smallest code that fails is `class Foo {}; export {Foo}`, though I think generators also trigger the error:

```sh
$ npm run build

> test@1.0.0 build /Users/carlos/Dropbox/Development/test
> rollup -c

Bad sourcemap
Error: Bad sourcemap
    at Object.Link.traceSegment (/Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:75:26)
    at /Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:31:27
    at Array.forEach (native)
    at /Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:29:9
    at Array.map (native)
    at Object.Link.traceMappings (/Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:26:34)
    at collapseSourcemaps (/Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/utils/collapseSourcemaps.js:102:37)
    at Bundle.render (/Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/Bundle.js:280:11)
    at Object.write (/Users/carlos/Dropbox/Development/test/node_modules/rollup/www/ROLLUP/rollup/src/rollup.js:62:32)
    at /Users/carlos/Dropbox/Development/test/node_modules/rollup/bin/runRollup.js:138:18
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

npm ERR! Darwin 15.4.0
npm ERR! argv "/Users/carlos/.nvm/versions/node/v5.7.0/bin/node" "/Users/carlos/.nvm/versions/node/v5.7.0/bin/npm" "run" "build"
npm ERR! node v5.7.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE
npm ERR! test@1.0.0 build: `rollup -c`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the test@1.0.0 build script 'rollup -c'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the test package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rollup -c
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs test
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls test
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/carlos/Dropbox/Development/test/npm-debug.log
```

I strongly suspect this has to do with the newly released [`babel-core@6.7.7`](https://github.com/babel/babel/blob/master/CHANGELOG.md#677-2016-04-20), as my project worked yesterday (with `babel-core@6.7.6`) but not today after I had to remove my `node_modules`.

**Comment by Rich-Harris** [comment]
I think this is the same as https://github.com/rollup/rollup/issues/618 – looking into it

**Comment by cprecioso** [comment]
Closed by rollup/rollup#619
